components/cache.rst

```diff
@@ -23,112 +24,117 @@ Installation
 
 .. include:: /components/require_autoload.rst.inc
 
-Cache (PSR-6) Versus Simple Cache (PSR-16)
-------------------------------------------
+Cache Contracts versus PSR-6
+----------------------------
 
 This component includes *two* different approaches to caching:
 
 :ref:`PSR-6 Caching <cache-component-psr6-caching>`:
-     A fully-featured cache system, which includes cache "pools", more advanced
-     cache "items", and :ref:`cache tagging for invalidation <cache-component-tags>`.
+     A generic cache system, which involves cache "pools" and cache "items".
 
-:ref:`PSR-16 Simple Caching <cache-component-psr16-caching>`:
-    A simple way to store, fetch and remove items from a cache.
+:ref:`Cache Contracts <cache-component-contracts>`:
```

Comment:
For this reference to work, you need to add this just before the `Cache Contracts` heading:

```rst
.. _cache-component-contracts:

Cache Contracts
---------------

...
```

Reply:
Thank you. I was wondering how that reference worked